src/components/LayoutHeader/Nav/LinkItems/styles.module.css

```diff
@@ -66,6 +78,18 @@
     }
   }
 
+  &.other {
```

Comment:
Here's what the popup looks like: 

![image](https://user-images.githubusercontent.com/43496356/153026740-0c6d7cf4-c6ea-4b35-9c1e-78d652e4c4d1.png)


Is it good enough for a first iteration? cc @yalozhkin